urumanifest/commit.py

```diff
@@ -46,6 +54,11 @@ def _hash_asset(args: Tuple[Path, Path]) -> Tuple[Path, str, int]:
     server_path, source_path = args
     # One day, we will not use such a vulnerable hashing algo...
     h = hashlib.md5()
+    if source_path.is_dir():
```

Comment:
Something I don't like here: The flag that this is an app bundle seems to be encoded later and isn't available yet. So I'm checking to see if the asset is a directory which is kind of gross. Ideally I'd only be checking the bitmask for the bundle property.